altair/api.py

```diff
@@ -233,6 +293,18 @@ def _shape_changed(self, name, old, new):
             self.shape._infer_type(self.parent.data)
 
 
+class Filter():
```

Comment:
Thanks, what is the time frame for the 0.8 release?

On Wed, Oct 14, 2015 at 7:21 PM, Kanit Wongsuphasawat <
notifications@github.com> wrote:

> In altair/api.py
> <https://github.com/ellisonbg/altair/pull/68#discussion_r42066894>:
>
> > @@ -233,6 +293,18 @@ def _shape_changed(self, name, old, new):
> >              self.shape._infer_type(self.parent.data)
> >
> >
> > +class Filter():
>
> I'm talking from a vega-lite syntax perspective.
>
> After 0.8 release, the top-level filter will become data.filter.
>
> Also, we are using Vega expression instead of the old predicate object.
>
> Here is the corresponding PR – vega/vega-lite#603
> <https://github.com/vega/vega-lite/pull/603>.
>
> For other syntax changes in the pending 0.8, please take a look at
> vega/vega-lite#595 <https://github.com/vega/vega-lite/pull/595>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ellisonbg/altair/pull/68/files#r42066894>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com